app/models/ontology.rb

```diff
@@ -274,6 +275,23 @@ def formality_levels
       where(ontologies: {id: self})
   end
 
+  def file_deleted?(commit_oid = nil)
+    !has_file?(commit_oid)
+  end
+
+  # alias_method doesn't work for this one.
+  def has_file?(commit_oid = nil)
+    has_file(commit_oid)
+  end
+
+  def has_file(commit_oid = nil)
```

Comment:
No thanks. The method right above this one takes care of that. Here, I overwrite the attribute getter.